### DISCUSSION TITLE
Trigger workflow event when PR is removed from the merge queue

### DISCUSSION THREAD

**Discussion by traviskroberts** [community/community#89919] [Actions]
### Select Topic Area

Product Feedback

### Body

It would be very helpful if we could trigger a Github action workflow when a pull request is removed from the merge queue. I can see that there is [webhook support](https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=dequeued#pull_request) for the `dequeued` type on the `pull_request` event, but the `pull_request` event for workflows does not support this type.

Are there any plans to add the `dequeued` (and `enqueued`) types to the `pull_request` event that triggers workflows?

**Comment by neebs12** [comment]
Yes! This would be great! At the moment, our developers are not informed when their PRs are removed from the queue, so this would be quite useful 🙏

**Comment by prpetten** [comment]
Yeah, this would be very useful to be exposed to workflows.  I'd like to add and remove a label to PRs based on a PR being "enqueued" and "dequeued".
